### DISCUSSION TITLE
Is lndg on telegram, slack or twitter?

### DISCUSSION THREAD

**Discussion by cliffpeskin** [cryptosharks131/lndg#163] [General]
Hi,

I recently installed lndg and would love to chat with others who are using it (I have many questions). Is there a telegram or slack channel?

UPDATE: I just found CryptoSharks on Twitter and this thread in particular is enormously helpful in getting started with LNDg https://twitter.com/CryptoSharks131/status/1490409704735973377?s=20&t=leBOf4pAbYSPcsip-6surg

**Comment by cryptosharks131** [comment]
TG: https://t.me/+cJj67U80jghkNzNh
